python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -499,28 +600,27 @@ def is_started(self):
         return not isinstance(self.worker_group, InactiveWorkerGroup)
 
     @property
-    def latest_run_dir(self) -> Optional[Path]:
-        """Path to the latest run directory."""
-        if self._run_id > 0:
-            run_dir = Path(f"run_{self._run_id:03d}")
-            return construct_path(run_dir, self.logdir)
-        else:
-            return None
+    def latest_run_dir(self):
+        return self.checkpoint_manager.latest_run_dir
```

Comment:
Is this still needed since it's created in the `Trainer` now?

Reply:
removed!